### DISCUSSION TITLE
Changing opacity, size of carbohydrates symbols and animate the structure (spin) programmatically?

### DISCUSSION THREAD

**Discussion by sujeetvkulkarni** [molstar/molstar#738] [Q&A]
Hi,

I am loading pdb files generated on Glycam-Web into the Mol* viewer. Is there a way to change opacity, size of 3D-SNFG (carbohydrate) symbols and animate the structure programmatically? 

Thanks,

**Comment by gitoliver** [comment]
We are also interested in the answer to this.

**Comment by sujeetvkulkarni** [comment]
Code for changing opacity, size of carbohydrates symbols programmatically - 
https://github.com/glygener/glygen-frontend/wiki/Changing-opacity,-size-of-carbohydrates-symbols
